### PR TITLE
Still on LC25

### PR DESCRIPTION
**Pull request by Grimlock123**
I am still working on LC25. I am receiving an error message that reads as "TypeError: Cannot read property 'next' of null" on line 60. I have LC 206 finished and will upload later in the day.

### CODE REVIEW

File: mouhamadouBarry/assignments/reverseLinked/lc25/lc25.js [hunk 1]
```diff
@@ -3,16 +3,39 @@
...
+    head - points to the 
+    size -  size of the linked list
+
+  Logic:
+    use the for loop to iterate through the linked list 
+
+    The head pointer assigned to the end of the linked
+    Then go in reverse order
+
+[[3,2,1],[4,5,6],[7,8]]
```
Comment (knrodriguez) [inline mouhamadouBarry/assignments/reverseLinked/lc25/lc25.js:24]: Try to diagram/write out the array with each sub-list reversal. It may help the understanding.

File: mouhamadouBarry/assignments/reverseLinked/lc25/lc25.js [hunk 2]
```diff
@@ -3,16 +3,39 @@
 // Given the head of a LinkedList and a number ‘k’, reverse every ‘k’ sized sub-list starting from the head.
 
 // If, in the end, you are left with a sub-list with less than ‘k’ elements, reverse it too.
-
+/*
```
Comment (knrodriguez) [inline mouhamadouBarry/assignments/reverseLinked/lc25/lc25.js:6]: Also add some edge cases and test cases to match.

**Review by knrodriguez — APPROVED** [review]
